docs/src/main/paradox/errorhandling.md

```diff
@@ -28,6 +28,11 @@ Scala
 Java
 : @@ snip [stopping](../../test/java/sample/javadsl/StreamWrapperActor.java) { #errorHandlingSupervisor }
 
+In order to handle **Consumer fails due to WakeupExceptions** while using the ClosedGraph.
+
+Java
+: @@ snip [stopping](../../test/java/sample/javadsl/RunnableGraphStreamWrapperActor.java) { ##errorHandlingConsumerFailsWakeupExceptions }
```

Comment:
Thank you 👍  I hope I fixed it. Unfortunately, I am not a Scala expert to add this example!